### PR TITLE
Ocd 2781 - Improve Reports responsiveness - page load - Product/Version Activity

### PR DESCRIPTION
**Pull request by tmy1313**
Contains commits related to OCD-2755

### CODE REVIEW

File: PENDING_RELEASE_NOTES.md [hunk 1]
```diff
@@ -2,13 +2,17 @@
...
+* Shrink main header navigation on page scroll
+* Show "history popup" on all Listings
+* Updated security for edit/split/merge of developers, products, and versions
 * Use /collections/decertified-developers endpoint instead of deprecated /decertifications/developers.
 * Show "history popup" on all Listings
 * Updated security for edit/split/merge of developers, products, and versions
 * Updated text and column headings on the Decertified Products and Inactive Certificates pages.
 * Updated language on Charts page.
-* Shrink main header navigation on page scroll
+* Updated produc and version activity reports to use activity metadata
```
Comment (andlar) [inline PENDING_RELEASE_NOTES.md]:
```suggestion
* Updated product and version activity reports to use activity metadata
```

File: src/app/pages/reports/products.component.js [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+                startDate: new Date(),
+                endDate: new Date(),
+            };
+            this.activityRange.startDate.setDate(this.activityRange.endDate.getDate() - this.activityRange.range + 1); // offset to account for inclusion of endDate in range
+            this.filename = 'Reports_' + new Date().getTime() + '.csv';
+            this.filterText = '';
+            this.tableController = {};
+        }
+
+        $onChanges () {
```
Comment (andlar) [inline src/app/pages/reports/products.component.js]: `$onChanges` takes a changes object. If you're just using this to search on init then this should be `$onInit`. If you're using it on change then there should be some logic that handles the changes

File: PENDING_RELEASE_NOTES.md [hunk 3]
```diff
@@ -1,17 +1,10 @@
 
-## Version 10.19.0
+## Version 10.20
```
Comment (andlar) [inline PENDING_RELEASE_NOTES.md]: Needs the patch version too

**Comment by andlar** [comment]
The thing that's left is removing the old Reports. chpladmin/chpl-website@bd104c0 is the commit where I moved the ACB report to the new page. It looks a lot like this PR, but this PR doesn't have all of the "removals" from the admin/reports/reports js & html files

File: PENDING_RELEASE_NOTES.md [hunk 4]
```diff
@@ -1,8 +1,9 @@
 
-## Version 10.19.0
+## Version 10.20.0
```
Comment (kekey1) [inline PENDING_RELEASE_NOTES.md]: The last release was 10.18 so I think this version should not have been changed and should remain 10.19 (for now).

**Pull request merged**